.github/workflows/build.yml

```diff
@@ -31,6 +42,19 @@ jobs:
           ${{ runner.os }}-
         path: ${{ steps.setup-haskell.outputs.cabal-store || '~/.cabal/store' }}
 
+    # FIXME: Find better approach to attain vendor dependencies of spectrometer.
+    # Since spectrometer's distribution does not include required extra-source-files,
+    # to successfully build spectrometer test project, mock vendor dependencies.
```

Comment:
We will probably never need these in integration tests.  These tools fulfill a contract, and we can only test that we uphold our end of the contract.  If we ever test these, it will be in end-to-end testing.  I don't think this is a FIXME, it's just "We don't need these, but can't build without them, so we mock them."